lib/lobbyist/v2/communication_campaign.rb

```diff
@@ -52,6 +52,10 @@ def self.search(params = {})
       def self.charge(id, params = {})
         create_from_response(post("/v2/campaigns/#{id}/charge.json", params))
       end
+
+      def self.clone_campaign(id, company_id)
```

Comment:
don't think we need company_id and it isn't used in the endpoint anyway.

Reply:
Removed.